portfolio/src/main/webapp/script.js

```diff
@@ -15,12 +15,55 @@
 /**
  * Adds a random greeting to the page.
  */
+
+
+ async function showServerHello() {
+  const responseFromServer = await fetch('/hello');
+  //const textFromResponse = await responseFromServer.text();
+  const hello = await responseFromServer.json();
+
+
+
+  const helloContainer = document.getElementById('hello-container');
+  helloContainer.innerHTML='';
+
+
+ 
+  helloContainer.appendChild(
+      createListElement('Movie #1: ' + hello.movie1));
+  helloContainer.appendChild(
+      createListElement('Movie #2: ' + hello.movie2));
+  helloContainer.appendChild(
+      createListElement('Movie #3: ' + hello.movie3));
+
+   
+  // Add it to the page.
```

Comment:
This comment looks like it belongs above the lines where you append list elements.

Reply:
I deleted it. It was in the wrong place.